Resources/Prototypes/_SpaceCats/Entities/Structures/Machines/Computers/book_printer.yml

```
@@ -0,0 +1,119 @@
+        node: machineFrame
+      - !type:DoActsBehavior
+        acts: ["Destruction"]
+  - type: ItemSlots
+    slots:
+       bookSlot:
+        whitelistFailPopup: book-printer-component-cannot-put-entity-message
+        whitelist:
+          tags:
+          - Book
+       cartridgeSlot:
+        whitelist:
```

yamllint (1.35.1)

[warning] wrong indentation: expected 9 but found 8

(indentation)